interface/reports/dictation_review.php

```diff
@@ -0,0 +1,316 @@
+    <?php //echo date("d F Y", strtotime($form_from_date)) ." &nbsp; to &nbsp; ". date("d F Y", strtotime($form_to_date)); ?>
+
+</div>
+
+<form name='theform' method='post' action='dictation_review.php' id='theform' onsubmit='return top.restoreSession()'>
+
+    <div id="report_parameters">
+
+        <input type='hidden' name='form_refresh' id='form_refresh' value=''/>
+        <input type='hidden' name='form_refresh1' id='form_refresh1' value=''/>
+
+        <table>
+            <tr>
+                <td width='410px'>
+                    <div style='float:left'>
+
+                        <table class='text'>
+                            <tr>
+                                <td class='control-label'>
+                                    <?php xl('From','e'); ?>:
+                                </td>
+                                <td class='control-label'>
+                                    <input type='text' name='form_from_date' id="form_from_date"
+                                           class='datepicker form-control'
+                                           size='10' value='<?php echo attr($from_date) ?>'
+                                           title='yyyy-mm-dd'>
+                                </td>
+                                <td class='control-label'>
+                                    <?php xl('To','e'); ?>:
+                                </td>
+                                <td>
+                                    <input type='text' name='form_to_date' id="form_to_date"
+                                           class='datepicker form-control'
+                                           size='10' value='<?php echo attr($to_date) ?>'
+                                           title='yyyy-mm-dd'>
+                                </td>
+                            </tr>
+                        </table>
+
+                    </div>
+
+                </td>
+                <td align='left' valign='middle' height="100%">
+                    <table style='border-left:1px solid; width:100%; height:100%' >
+                        <tr>
+                            <td>
+                                <div class="text-center">
+                                    <div class="btn-group" role="group">
+                                    <a href='#' class='btn btn-default btn-save' onclick='$("#form_refresh").attr("value","true"); $("#theform").submit();'>
+					<span>
+						<?php xl('Submit','e'); ?>
+					</span>
+                                    </a>
+
+                                    <?php if ($_POST['form_refresh']) { ?>
+                                        <a href='#' class='css_button' onclick='window.print()'>
+						<span>
+							<?php xl('Print','e'); ?>
+						</span>
+                                        </a>
+                                        <a href='#' class='css_button' onclick='$("#form_refresh1").attr("value","true"); $("#theform").submit();'>
```

Comment:
check out the other reports in development codebase for which class to use for this Save (rec using the btn-transmit class since you use the btn-save for the Submit button above)

Reply:
I did not realize how old this feature was until I brought it into the new code base. I did a lot of cleaning up before submitting it for your review. Of course, there is more to do.

Reply:
Agreed that you already did a good job in getting it mostly up to current standards! Just a little more minor stuff to do in this regard.